control/statesp.py

```diff
@@ -70,7 +70,7 @@
 
 # Define module default parameter values
 _statesp_defaults = {
-    'statesp.use_numpy_matrix': True,
+    'statesp.use_numpy_matrix': False,
```

Comment:
Please don't change the default setting here. We have a road map in mind when this should happen and it should be independent of the support for complex types.

Reply:
That said, actually #438 takes [complex inputs into account](https://github.com/python-control/python-control/pull/438/files#diff-b5aac033a6dd07653e82d9f5a40afaa775010c2d2d5af8beeb159487b566df72)  already!

Reply:
Ok, I will change this back

Reply:
> That said, actually #438 takes [complex inputs into account](https://github.com/python-control/python-control/pull/438/files#diff-b5aac033a6dd07653e82d9f5a40afaa775010c2d2d5af8beeb159487b566df72) already!

Oh, I see, should I close my PR?